lang/src/main/antlr4/org/kaazing/k3po/lang/parser/v2/Robot.g4

```diff
@@ -41,7 +41,7 @@ acceptableNode
     ;
 
 connectNode
-    : k=ConnectKeyword (AwaitKeyword barrier=Name ConnectKeyword)? connectURI=location  streamableNode+
+    : k=ConnectKeyword (AwaitKeyword barrier=Name ConnectKeyword)? (OptionKeyword TransportKeyword transport=TextLiteral ConnectKeyword)? connectURI=location  streamableNode+
     ;
```

Comment:
Recommend making this generic syntax for any option.
```
connectNode
   : k=ConnectKeyword (AwaitKeyword barrier=Name ConnectKeyword)? (OptionKeyword optionName=ConnectOptionName optionValue=TextLiteral ConnectKeyword)? connectURI=location  streamableNode+
```
...
```
ConnectOptionName
  : 'transport'
  ;
```
Other connect options would then be added to the grammar here in future.